### DISCUSSION TITLE
Example Packages with typedi?

### DISCUSSION THREAD

**Discussion by naydichev** [MichalLytek/type-graphql#1351] [Q&A]
Hello,

I'm trying to use `type-graphql` with `typedi` and I'm running into a variety of problems. I was hoping that someone out there could point me to an example repository that uses the two packages together correctly so that I may try to figure out what my mistakes are.

For reference, all of my classes are getting the `ContainerInstance` as the first argument, and nothing else. The `ContainerInstance` contains all of the classes that I've decorated with `@Service()`, but `typedi` just isn't passing them as arguments for some reason.

Anyway, I'm kind of stuck and not sure why this isn't working - so any pointers would be appreciated.

**Comment by Cry0nicS** [comment]
I have commented on a similar discussion here https://github.com/MichalLytek/type-graphql/issues/1252#issuecomment-1185717024

You should find a working example as well. My example is with TypeORM, but you can use it for anything else you'd like to inject.

Hope it helps.
